Packs/CommonTypes/ReleaseNotes/3_1_31.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges": true, "breakingChangesNotes": null}
```

Comment:
why are we not adding the breakingChangesNotes?

Reply:
If it is null, by the BC design it will just show the whole RN file so i didnt add specific text